demisto_sdk/commands/upload/README.md

```diff
@@ -8,7 +8,7 @@ and `DEMISTO_API_KEY` environment variable should contain a valid Cortex XSOAR/X
 **Note:** Cortex XSIAM Base URL should be retrieved from XSIAM instance -> Settings -> Configurations -> API Keys -> `Copy URL` button on the top rigth corner, and not the browser URL.
```

Comment:
Please also add that it's the same process for XSOAR 8.X.X (verify first that it's true)

Reply:
same process